examples/gno.land/p/demo/zteams/teams.gno

```diff
@@ -0,0 +1,262 @@
+package zteams
+
+import (
+    "std"
+    "gno.land/p/demo/avl"
+    "gno.land/p/demo/users"
+)
+
+// Team represents a team with its details and roles.
+type Team struct {
+    Address     std.Address
+    TeamName    string
+    TeamProfile string
+    Owner       std.Address
+    Admins      map[string]bool // Changed to string keys
+    Disbursers  map[string]bool // Changed to string keys
```

Comment:
How about chainging `Admins` and `Disbursers` types to `map[std.Address]bool` to reduce string conversions?

Reply:
I used different types than `string` in maps and avl.Trees before, but I randomly ran into the error: `unexpected object with identifier...`. This didn't happen when I used strings. I reckon that error, which sits at the vm level, is not solved yet. So, I would rather stay on the safe side. It's an easy fix, once the vm bug is fixed.